pkg/graveler/graveler.go

```diff
@@ -2551,6 +2551,9 @@ func (g *Graveler) Revert(ctx context.Context, repository *RepositoryRecord, bra
 		}
 		// merge from the parent to the top of the branch, with the given ref as the merge base:
 		metaRangeID, err := g.CommittedManager.Merge(ctx, repository.StorageNamespace, branchCommit.MetaRangeID, parentMetaRangeID, commitRecord.MetaRangeID, MergeStrategyNone)
+		if (metaRangeID == branchCommit.MetaRangeID) && !commitParams.AllowEmpty {
```

Comment:
Missed this one in previs PR - we can't check the return value before we verified the error first.